blocks/api/raw-handling/utils.js

```diff
@@ -31,6 +31,11 @@ const inlineWhitelist = {
 	br: {},
 };
 
+const inlineBlockWhiteList = {
```

Comment:
I'm a bit concerned that all these lists and moving parts in `raw-handling` grow hard to keep track of, e.g. "inline" vs "inline wrapper" vs. "inline block". Docs in this file would help.